manifests/pipecd/values.yaml

```diff
@@ -30,6 +30,8 @@ gateway:
 server:
   image:
     repository: ghcr.io/pipe-cd/pipecd
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
```

Comment:
As mentioned in template file, this config should be under `image` block (same as `repository`)